### PR TITLE
fix https://github.com/NanoMeow/QuickReports/issues/4778

### PR DESCRIPTION
**Pull request by Yuki2718**
https://github.com/NanoMeow/QuickReports/issues/4778
Fixed in Base as there is already an entry for its sister site (`9tsu.net`). `ad-vortex.com` is only on 9tsu.

### CODE REVIEW

**Comment by Yuki2718** [comment]
@ryanbr I wonder if
```
||ad-vortex.com^
##.ads-system
```
can be added to EL.

File: EnglishFilter/sections/adservers.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@
 !! https://a.bestcontentitem.top/warp/*?r=
 !
 ||poemsdeucewee.com^
+||ad-vortex.com^
```
Comment (zloyden) [inline EnglishFilter/sections/adservers.txt:9]: We already have this rule in Easylist.

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -855,7 +855,8 @@ kotaku.com##.js_movable_ad_slot
 forbes.com##fbs-ad + p > br
 swordz.io##div[style="width:300; height: 250; background-color: #000"]
 disasterscans.com##div[id^="media_image-"].no-icon.heading-style-1
-9tsu.net##.ads-system
+9tsu.net,miomio.us##.ads-system
+9tsu.net,miomio.us##.cactus-sidebar-content > .widget_text
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:859]: I don't see any other languages except Japanese. So, please move it to the AdGuard Japanese filter -> specific section.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/adservers.txt (hunk 1 above)
Comment (Yuki2718) [reply EnglishFilter/sections/adservers.txt:9]: Then that must be added after my commit.

File: EnglishFilter/sections/specific.txt (hunk 2 above)
Comment (Yuki2718) [reply EnglishFilter/sections/specific.txt:859]: Okay.

**Comment by zloyden** [comment]
@Yuki2718 if you can't make changes, please tell me. I can do it directly.

**Comment by Yuki2718** [comment]
> @Yuki2718 if you can't make changes, please tell me. I can do it directly.

Please do so, that'll be faster and mine is likely to cause conflict.

**Comment by zloyden** [comment]
Added it to our filters, thank you!

**Pull request closed without merging**